packages/react/package.json

```diff
@@ -34,7 +34,7 @@
     "lint": "yarn lint:javascript && yarn lint:stylelint",
     "lint:javascript": "eslint --ext .jsx --ext .js .",
     "lint:stylelint": "stylelint './src/**/*.scss' --syntax scss --ignorePath .gitignore --custom-formatter ./config/stylelint/sassMsgFormatter.js",
-    "pre-commit": "yarn svg-convert && yarn lint-staged && yarn test:base",
+    "pre-commit": "yarn svg-convert && yarn lint-staged && yarn reports:clean && yarn test:base && yarn test:e2e && yarn reports",
```

Comment:
whew. our pre-commits are going to take _a lot_ of time as the e2e test coverage grows...

Reply:
Yes, I was thinking this. I am not sure if it's more annoying to wait for test to pass before pushing up your code or waiting on the actions to run which  may be in a queue and then having to push up more code if there are failures. Thoughts?

Reply:
It's definitely a double-edged sword. There are definitely times I don't break changes up into small commits because the pre-commit takes so long. This is primarily because I view `--no-verify` as dirty. However, since we're limited on the number of actions that can run at a time, I prefer the more immediate feedback. Maybe it would be nice to move these to a `pre-push` hook, so that local dev is faster, but ensures everything is clean before it's pushed up?